.github/scripts/end2end/deploy-metadata.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -exu
+
+. "$(dirname $0)/common.sh"
+
+# create a separate namespace for metadata
+kubectl create namespace metadata
+
+# clone the metadata repository
+git init metadata
+cd metadata
+git fetch --depth 1 --no-tags https://${GIT_ACCESS_TOKEN}@github.com/scality/metadata.git
+git checkout FETCH_HEAD
+
+# install metadata chart in a separate namespace
+cd helm
+helm dependency update cloudserver/
+helm install -n metadata \
+    --set metadata.persistentVolume.storageClass='' \
+    --set metadata.sproxyd.persistentVolume.storageClass='' \
+    s3c cloudserver/
+
+# wait for the repds to be created
+kubectl -n metadata rollout status --watch --timeout=300s statefulset/s3c-metadata-repd
+# wait for all repd pods to start serving admin API ports
+wait_for_all_pods_behind_services metadata-repd metadata "91*" 60
+
+# current chart uses an old version of bucketd that has issues reconnecting to the repd
+# when bucketd is started first. Restarting bucketd after repd is ready.
+kubectl -n metadata rollout restart deployment/s3c-metadata-bucketd
+# wait for the bucketd pods to be created
+kubectl -n metadata rollout status --watch --timeout=300s deploy/s3c-metadata-bucketd
+# wait for all bucketd pods to start serving port 9000
+wait_for_all_pods_behind_services metadata-bucketd metadata 9000 60
+
+# manually add "s3c.local" to the rest endpoints list as it's not configurable in the chart
```

Comment:
why do we need `s3c.local` ?

i did not need such patch on my artesca cluster (in namespace `default` i think), just installed the chart and created the artesca location pointed at `s3c-cloudserver.default.svc.cluster.local`

Reply:
Cloudserver responds with an error when the host used is not declared in its config file. I've tried with the k8s service endpoint and it doesn't work.

Reply:
maybe this comes from the namespace ("metadata") you deploy to, which may not be taken into account?
in my tests this is using the default namespace, with no issue...

Reply:
No even with the default namespace i got the same behaviour, which to me, is the expected behaviour. The normal cloudserver we deploy in Zenko has all the k8s endpoints in its config.